app.base.json

```diff
@@ -26,14 +26,11 @@
     "themeColor": "#ec0000"
   },
   "language": {
-    "avgDisclaimer": "Signalen verwerkt uw persoonsgegevens op een zorgvuldige en veilige manier in overeenstemming met de geldende wet- en regelgeving, waaronder de Algemene Verordening Gegevensbescherming. Lees meer hierover op <a href='##AVG_DISCLAIMER_URL##'>Privacyverklaring Meldingen openbare ruimte en overlast</a>.",
     "consentToContactSharing": "Ja, ik geef Signalen toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
     "district": "Wijk",
-    "footer1": "Bel ons dan",
-    "footer2": "op werkdagen van 9.00 tot 17.00 uur.",
     "headerTitle": "",
-    "helpText": "Bereikbaar van maandag tot en met vrijdag van 08:00 tot 18:00 uur.",
-    "helpTextHeader": "Hulp nodig: Bel het telefoonnummer [14020](tel:14020)",
+    "helpText": "",
+    "helpTextHeader": "",
```

Comment:
You can use the same values as in `config.amsterdam.json`. With the phone number as in this file.

Reply:
✔️